stacks/root.yml

```diff
@@ -310,6 +310,26 @@ Resources:
       TemplateURL: !Sub ${Constants.TemplateUrlBase}/stacks/shared-announce.yml
       TimeoutInMinutes: 10
 
+  SharedKinesisStack:
+    Type: AWS::CloudFormation::Stack
+    Properties:
+      NotificationARNs:
+        - !Ref CloudFormationNotificationsSnsTopicArn
+      Parameters:
+        RootStackName: !GetAtt Constants.RootStackName
+        RootStackId: !GetAtt Constants.RootStackId
+        EnvironmentType: !Ref EnvironmentType
+      Tags:
+        - { Key: prx:meta:tagging-version, Value: "2021-04-07" }
+        - { Key: prx:cloudformation:stack-name, Value: !Ref AWS::StackName }
+        - { Key: prx:cloudformation:stack-id, Value: !Ref AWS::StackId }
+        - { Key: prx:cloudformation:root-stack-name, Value: !GetAtt Constants.RootStackName }
+        - { Key: prx:cloudformation:root-stack-id, Value: !GetAtt Constants.RootStackId }
+        - { Key: prx:ops:environment, Value: !Ref EnvironmentType }
+        - { Key: prx:dev:application, Value: Common }
+      TemplateURL: !Sub ${Constants.TemplateUrlBase}/stacks/shared-kinesis.yml
+      TimeoutInMinutes: 10
```

Comment:
I would make this timeout much higher, like `60` or `999`. If CFN every tried to like reshard, fail, and roll back, I bet that would take a lot longer than 10 mintues

Reply:
Does resharding take time?  I feel like every time I've done that in the web UI, it's instantaneous.  (Whatever's already in Kinesis stays in whatever shard - it only impacts new stuff).

Maybe it's different if you remove a shard?  Yeah, I'll change to 60.